### ISSUE TITLE
Make it clearer on the website that React D3 is deprecated / no longer maintained

### ISSUE THREAD

**Issue by sihil** [react-d3/react-d3.github.io#17]
The note in the github description is not enough to ensure that developers do not inadvertently pick up this abandoned library. I was going to try and make a PR but I can't get this to build under OSX.

I suggest a clear banner across the top of all pages saying that this is no longer maintained.

For those that come after me I've found this useful: https://oli.me.uk/2015/09/09/d3-within-react-the-right-way/

**Comment by svet93** [comment]
I totally agree. I just spent 3 hours messing around with the library before realizing it's no longer maintained...

**Comment by Ding-Fan** [comment]
Same here. And there would be more people fix the /gallery issue if they didn't read the pull request list.

**Comment by dabstractor** [comment]
That web site is just so flashy it makes it feel right.
